frontend/lib/src/features/transaction_history/transaction_history.dart

```diff
@@ -25,29 +25,28 @@ class TransactionHistory extends StatelessWidget {
     TransactionController controller = Get.put(TransactionController());
     final arguments =
         ModalRoute.of(context)!.settings.arguments as ScreenArguments;
+    return Obx(
```

Comment:
controller.getTransactions is a `function`
we use OBX/GETX widget when its child is managing observables

this causes the whole page to reload which is a very weird sideeffect,

this is wrong implementation of Obx